sqld/src/lib.rs

```diff
@@ -99,9 +112,20 @@ where
         handles.push(handle);
     }
 
-    while let Some(res) = handles.next().await {
-        res??;
-    }
+    Ok(())
+}
+
+/// nukes current DB and start anew
+async fn hard_reset(config: &Config, mut handles: Handles) -> anyhow::Result<()> {
+    tracing::error!("received hard-reset command: reseting replica.");
+
+    tracing::info!("Shutting down all services...");
+    handles.iter_mut().for_each(|h| h.abort());
+    while let Some(_) = handles.next().await {}
```

Comment:
We don't want to report potential errors returned from handles?

Reply:
the error is always going to be an abort